src/main/webapp/app/modules/provider/all-records.tsx

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
+import React, { ComponentClass, StatelessComponent } from 'react';
 import { getAllProviderRecords } from './provider-record.reducer';
 import { connect } from 'react-redux';
 import { Col, Row, Progress, Modal } from 'reactstrap';
 import _ from 'lodash';
 import RecordCard from 'app/modules/provider/record/record-card';
-import { IPaginationBaseState, Translate } from 'react-jhipster';
+import { IPaginationBaseState, Translate, translate } from 'react-jhipster';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import SortSection from 'app/modules/provider/sort-section';
 import { getSearchPreferences, PROVIDER_SORT_ARRAY, setProviderSort } from 'app/shared/util/search-utils';
 import ReactGA from 'react-ga';
 import ButtonPill from './shared/button-pill';
 import FilterCard from './filter-card';
 import MediaQuery from 'react-responsive';
+import { withScriptjs, withGoogleMap, GoogleMap, Marker } from 'react-google-maps';
+import { GOOGLE_API_KEY } from 'app/config/constants';
+
+declare global {
```

Comment:
I don't see any usage of this. Is this necessary?

Reply:
yes it is for the new window.google... (it would be fine without it but its for eslint)